### PR TITLE
deployment: adding ember-cli-deploy-cloudfront for invalidations

### LINKED ISSUE #20: Fix the deployment of docs.cardstack.com
This is a **very** low priority issue and should be limited to 1 hour of time. If it takes any longer than that it should be removed from the project and left in the backlog :+1:

We have recently released documentation for the grants system on the documentation site. But when you load the page you will notice that the `grants` Table of Contents item appears at first and then disappears.

The cause of this is that the data json items are not cleared properly from Cloudfront https://docs.cardstack.com/content/release/pages.json

There is some configuration to perform an invalidation on Cloudfront in the ember-cli-deploy config but it clearly is not working right: https://github.com/cardstack/cardstack-documentation/blob/master/config/deploy.js#L34

I have a feeling that either the right ember-cli-deploy plugin that does cloudfront invalidations isn't installed or the invalidation actually needs to be explicit about the files that it needs to clear.

### PR DESCRIPTION
**Pull request by mansona**
Fixes #20 

Part of the config for this was already added but the required ember-addon wasn't installed 👍 

This mostly needs to be a "test in production" situation to see if it does what it is supposed to. Once the PR is merged we will see the result of the operation in the Travis log 👍

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -5800,6 +5800,27 @@
         "rsvp": "^3.5.0"
       }
     },
+    "ember-cli-deploy-cloudfront": {
```
Comment (Turbo87) [inline package-lock.json]: OMG it's using a `package-lock.json` 😱

File: package-lock.json [hunk 2]
```diff
@@ -9556,6 +9590,7 @@
           "version": "1.4.0",
           "bundled": true,
           "dev": true,
+          "optional": true,
```
Comment (Turbo87) [inline package-lock.json]: these changes look unrelated?

File: config/deploy.js [hunk 3]
```diff
@@ -33,6 +33,8 @@ module.exports = function(deployTarget) {
     ENV["s3-index"].region = 'ap-southeast-1';
     ENV.cloudfront = {
       distribution: 'E2TZ3PLCL45IAQ',
+      accessKeyId: process.env.AWS_SECRET_ACCESS_KEY,
```
Comment (Turbo87) [inline config/deploy.js]: is that a copy-paste error or is it supposed to use the same env var for both properties?

Comment (mansona) [reply config/deploy.js]: thanks, it's a copy 🍝 error 🙈

File: package-lock.json (hunk 1 above)
Comment (mansona) [reply package-lock.json]: yea 😂

File: package-lock.json (hunk 2 above)
Comment (mansona) [reply package-lock.json]: I would leave them in anyway 👍
